src/mixins/filing-template-mixin.ts

```diff
@@ -1266,55 +1293,53 @@ export default class FilingTemplateMixin extends DateMixin {
   }
 
   /**
-   * Parses an entity snapshot into the store.
+   * Parses an entity snapshot into the store. Used when there isn't a draft.
    * @param entitySnapshot the latest entity snapshot
    */
   parseEntitySnapshot (entitySnapshot = this.getEntitySnapshot): void {
     // store Entity Snapshot
     this.setEntitySnapshot(entitySnapshot)
 
     // store Folio Number
-    this.setFolioNumber(entitySnapshot.authInfo.folioNumber || '')
+    this.setFolioNumber(entitySnapshot?.authInfo?.folioNumber || '')
 
     // store Entity Type
-    this.setEntityType(entitySnapshot.businessInfo.legalType)
+    this.setEntityType(entitySnapshot?.businessInfo?.legalType || null)
 
     // store Business Information
-    this.setBusinessInformation({ ...entitySnapshot.businessInfo })
+    this.setBusinessInformation({ ...entitySnapshot?.businessInfo } || { ...EmptyBusinessInfo })
```

Comment:
This, and many other conditional chaining and fallbacks in here, are defensive coding -- should never happen, but it if does then at least THIS code won't crash (cause exception) and the store won't be correct.